.buildkite/pipeline.full.yml

```diff
@@ -29,7 +29,7 @@ steps:
       queue: macos-12-arm
     command: './gradlew clean assembleRelease check --scan'
 
-  - label: ':android: Minimal fixture end-to-end tests'
+  - label: ':browserstack: Minimal fixture end-to-end tests'
```

Comment:
The intention with these is so that we can see at a glance on the Buildkite results page which device farm a step runs on.  BrowserStack already have a custom emojicon, BitBar is on its way (so will just display as ":bitbar:" in the short term.